iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm

```diff
@@ -138,6 +139,9 @@ - (IBAction)login
       catch (exception const & ex)
       {
         LOG(LWARNING, ("Error login", ex.what()));
+        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
```

Comment:
Кто мне объяснит отличия в этих ключах статистики в вики? Я предполагаю, что ошиблись при формулировке задачи.

Нам нужен факт успеха или ошибка после входа через осм фб или гугл. Дополнительных смыслов вносить в события не вижу.

> On 25.3.2016, at 21.48, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm:
> 
> > @@ -138,6 +139,9 @@ - (IBAction)login
> >        catch (exception const & ex)
> >        {
> >          LOG(LWARNING, ("Error login", ex.what()));
> > +        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> Ключи для авторизации и регистрации всё равно различные, это ортогонально критерию facebook-google-osm Из доки, по которой делаем статистику: Editor_Auth_request (aloha & flurry): authentication process: auth request was sent type: string: facebook, google, osm Editor_Reg_request (aloha & flurry): registration process: reg process started on OSM site Editor_Auth_request_result: authentication process: auth process was finished is_success: boolean err_data: string Editor_Reg_request_result: registration process: reg process finished on OSM site is_success: boolean err_data: string
> …
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>